from http://lists.mysql.com/eventum-devel/141
2. Then the regular expression to match cvs commit messages on issue
numbers doesn't work. It will accept almost anything with a number in
it.

--- eventum-1.4/misc/scm/process_cvs_commits.php~	2005-02-07 16:24:57.000000000 +0200
+++ eventum-1.4/misc/scm/process_cvs_commits.php	2005-02-07 16:27:03.000000000 +0200
@@ -68,7 +68,7 @@
 $commit_msg = substr($input, strpos($input, 'Log Message:')+strlen('Log Message:')+1);
 
 // parse the commit message and get the first issue number we can find
-$pattern = "/[issue|bug] ?:? ?#?(\d+)/i";
+$pattern = "/(?:issue|bug) ?:? ?#?(\d+)/i";
 preg_match($pattern, $commit_msg, $matches);
 
 if (count($matches) > 1) {
